packages/cloud-cognitive/src/components/ComboButton/_combo-button.scss

```diff
@@ -98,6 +98,20 @@
     display: none;
   }
 
+  .#{$blockClass}__divider {
+    width: 100%;
+    height: 1px;
+    background-color: $border-subtle;
+  }
+
+  .#{$blockClass}__overflow-menu__divider::before {
```

Comment:
Is this still in use? May be leftover from an older implementation or is this intended as a divider when there’s also a danger button after? Either way, I don’t see this reflected in the current implementation.